code/style.css

```diff
@@ -120,15 +120,14 @@ main {
 }
 
 button {
-  background-color: #0026ff;
-  color: white;
+  background-color: rgb(38, 137, 230);
+  color: black;
```

Comment:
This is a small comment, but both hex and rgb are used for the colors. Maybe the ones where the color is typed out `color: black` alos could be named with hex just to try to do it more the same throughout the code?